cmd/verrazzano-operator/main_test.go

```diff
@@ -0,0 +1,174 @@
+		"kubeconfig":                   "",
+		"watchNamespace":               "",
+		"verrazzanoUri":                "",
+		"helidonAppOperatorDeployment": "",
+		"enableMonitoringStorage":      "",
+		"apiServerRealm":               "",
+	}
+	var expectedBoolFlags = map[string]bool{
+		"startController": true,
+	}
+
+	foundStringFlags := map[string]string{}
+	foundBoolFlags := map[string]bool{}
+
+	// GIVEN functions to "register" string and bool flags
+	stringVarFunc := func(p *string, name string, value string, usage string) {
+		foundStringFlags[name] = value
+	}
+	boolVarFunc := func(p *bool, name string, value bool, usage string) {
+		foundBoolFlags[name] = value
+	}
+
+	// WHEN its initFlags is called
+	initFlags(stringVarFunc, boolVarFunc)
+
+	// THEN assert that the expected command line flags are initialized
+	assert.Equal(t, len(expectedStringFlags), len(foundStringFlags))
+	assert.Equal(t, len(expectedBoolFlags), len(foundBoolFlags))
+	for expectedName, expectedValue := range expectedStringFlags {
+		assert.Equal(t, expectedValue, foundStringFlags[expectedName])
+	}
+	for expectedName, expectedValue := range expectedBoolFlags {
+		assert.Equal(t, expectedValue, foundBoolFlags[expectedName])
+	}
+}
+
+// assertNotExpected asserts that the rootRouter does NOT contain a match for the given
+// http method and path
+func assertNotExpected(t *testing.T, rootRouter *mux.Router, method string, path string) {
+	req, err := http.NewRequest(method, path, nil)
+	assert.Nil(t, err, "Unexpected error creating test http request for path "+path)
+	routeMatch := &mux.RouteMatch{}
+	assert.False(t, rootRouter.Match(req, routeMatch),
+		fmt.Sprintf("Unexpected Match found for '%s %s'", method, path))
+}
+
+// assertExpected asserts that the rootRouter contains a match for the given http method and path
+func assertExpected(t *testing.T, rootRouter *mux.Router, method string, path string) {
+	req, err := http.NewRequest(method, path, nil)
+	assert.Nil(t, err, "Unexpected error creating test http request for path "+path)
+	routeMatch := &mux.RouteMatch{}
+
+	// assert that the match for given request path and request method was found
+	assert.True(t, rootRouter.Match(req, routeMatch),
+		fmt.Sprintf("Matcher for '%s %s' not found", method, path))
+
+	assert.Nil(t, routeMatch.MatchErr,
+		fmt.Sprintf("Error matching path %s: %s", path, routeMatch.MatchErr))
+
+	fmt.Printf("Matched %s %s\n", method, path)
+}
+
+// getComplementaryMethods returns the list of all http methods that are NOT in the given list
+// e.g. if given {"GET", "PUT"}, then it returns a list of all http methods other than GET and PUT
+func getComplementaryMethods(methods []string) []string {
+	complementaryMethods := []string{}
+	for _, eachMethod := range allHTTPMethods {
+		if !sliceContains(methods, eachMethod) {
+			complementaryMethods = append(complementaryMethods, eachMethod)
+		}
+	}
+	return complementaryMethods
+}
+
+func sliceContains(theSlice []string, itemToFind string) bool {
```

Comment:
this function is in utils/utils.go, called Contains

Reply:
thanks. changed to use that and remove duplicate